topics/microbiome/tutorials/visualisation-ampvis/tutorial.md

```diff
@@ -0,0 +1,623 @@
+---
+layout: tutorial_hands_on
+
+
+title: Visualization workflows for metagenomic amplicon data using the Galaxy framework
```

Comment:
```suggestion
title: Divers and adaptable visualizations of metabarcoding data using ampvis2
```
* Rather tell what the tutorial is about

Reply:
done